### PR TITLE
Wagtail css + Permalink fixes

### PR DESCRIPTION
**Pull request by FilipLeitner**
## Description

Please give a short description of the changes you propose. Please use prose and try not to be too technical, if possible. Make sure to mention people that you think should know about the PR.

## Related issues or pull requests

Please list issues or pull requests that the changes you propose are related to. It does not matter if they are still open and/or unmerged, any link is appreciated.

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [x] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [x] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Review by fzadrazil — COMMENTED** [review]
The styling seems mostly ok, but buttons in measure or map swipe panel have still "wagtail theme" background. Not sure whether it is wrong or not. We could keep some of the portal colors even for the map.

Permalink keeps extent now, perfect. I have noticed that the map doesn't keep the basemap though.

**Comment by FilipLeitner** [comment]
Okay, I will resolve that as well.
...I would just stick with our styling because those wagtail theme effects don't seem to be really consistent. It takes a rather specific CSS rule to bypass both our .hsl encapsulation or the bootstrap itself. So sometimes the change goes through sometimes not.

File: projects/hslayers/src/css/hslayers-bootstrap.scss [hunk 1]
```diff
@@ -507,6 +543,8 @@ $list-group-item-padding-x: 1.25rem;
     }
 
     /* Next is needed on iPhone/Safari even if the position gets 
+    /* Next is needed on iPhone/Safari even if the position gets 
+  /* Next is needed on iPhone/Safari even if the position gets
```
Comment (jmacura) [inline projects/hslayers/src/css/hslayers-bootstrap.scss:547]: mistake?

Comment (FilipLeitner) [reply projects/hslayers/src/css/hslayers-bootstrap.scss:547]: Yes, thanks for spotting it. Will fix

**Review by fzadrazil — APPROVED** [review]

**Pull request merged**